2022q4/status.adoc

```diff
@@ -32,9 +32,12 @@ Thus just "status" has been chosen: this is correct as quarterly status reports
 
 The following link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=267813[email addresses have been created]:
 
-* the main contact address for the status report team is now <status@FreeBSD.org>. Mails sent to <quarterly@FreeBSD.org> will still reach the team, but you are encouraged to use the new address;
-* the email address for the status report submissions is now <status-submissions@FreeBSD.org>. Mails sent to <quarterly-submissions@FreeBSD.org> will still reach the team, but you are encouraged to use the new address;
-* the <quarterly-calls@FreeBSD.org> mailing list has been renamed to <status-calls@FreeBSD.org>. If you were already subscribed to <quarterly-calls@FreeBSD.org>, you do not need to resubscribe to <status-calls@FreeBSD.org>.
+* the main contact address for the status report team is now <status@FreeBSD.org>.
+Mails sent to <quarterly@FreeBSD.org> will still reach the team, but you are encouraged to use the new address;
+* the email address for the status report submissions is now <status-submissions@FreeBSD.org>.
+Mails sent to <quarterly-submissions@FreeBSD.org> will still reach the team, but you are encouraged to use the new address;
+* the <quarterly-calls@FreeBSD.org> mailing list has been renamed to <status-calls@FreeBSD.org>.
+If you were already subscribed to <quarterly-calls@FreeBSD.org>, you do not need to resubscribe to <status-calls@FreeBSD.org>.
```

Comment:
As in the comment above, I would add a bit of spaces for new lines within the same list element to have better lines alignment.

Reply:
Please see above <https://github.com/freebsd/freebsd-quarterly/pull/558/files#r1060067753> re: ambiguity arising from artificial indents.